interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: That is a good question as this is something I first learned. Git is a version control software that is a local repository while Github is a version control, cloudservice that is a remote repository. Github is also where your code is stored. 
 
-Researched answer:
+Researched answer: Git keeps track of your code history (inputting history into your terminal will show your last logs). Github is the management system to help manage your repositories and also is a place of storage for code. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: That is a great question, at this time I'd love to research that question more in order to provide the best possible full answer for you. 
```

Comment:
Very polite answer.  I would recommend seeing if you can solve a simpler problem, or pick some of the vocabulary used and talk briefly about that as well.  For instance, if you know Boolean is a true or false value, talk about that.  Focus less on what you don't know, and more on what you do know.